app/services/lambda_jobs/runner.rb

```diff
@@ -8,13 +10,31 @@ def initialize(job_class:, args:)
     end
 
     def run(&local_callback)
-      if LoginGov::Hostdata.in_datacenter? && AppConfig.env.aws_lambda_proofing_enabled == 'true'
-        aws_lambda_client.invoke(
-          function_name: function_name,
-          invocation_type: 'Event',
-          log_type: 'None',
-          payload: args.to_json,
-        )
+      if AppConfig.env.aws_lambda_proofing_enabled == 'true'
+        if LoginGov::Hostdata.in_datacenter?
+          aws_lambda_client.invoke(
+            function_name: function_name,
+            invocation_type: 'Event',
+            log_type: 'None',
+            payload: args.to_json,
+          )
+        else
+          Thread.new do
+            sleep AppConfig.env.aws_lambda_proofing_local_delay_in_seconds.to_i
+
+            job_class.handle(
+              event: args,
+              context: nil,
+            ) do |result|
+              Faraday.post(
+                args[:callback_url],
+                result.to_json,
+                'X-API-AUTH-TOKEN' => AppConfig.env.resolution_proof_result_lambda_token,
```

Comment:
Just realized....we can't hardcode the auth token like this, we'll need to update call sites to pass the correct token per job (resolution, address, document)

Reply:
> Just realized....we can't hardcode the auth token like this, we'll need to update call sites to pass the correct token per job (resolution, address, document)

Ah, good catch. I suppose we could add a second argument with the token it'd be expected to call back with. Or alternatively disable token verification in local development (probably inadvisable if the goal is more "realism").